docs/developer-docs/integrations/ledger/introduction_and_overview.md

```diff
@@ -29,12 +29,21 @@ ICRC-1 on the other hand is a standard, not a specific implementation of some le
 The ICP ledger uses `AccountIdentifiers` as its account representation. You can find a detailed description of `AccountIdentifier` [here](https://mmapped.blog/posts/13-icp-ledger#account-id). An`AccountIdentifier` essentially is a hash of the ICRC-1 `Account`. An `Account` can thus be converted into an `AccountIdentifier` but not the other
 way around. This provides the benefit of providing a certain degree of anonymity to the ICP ledger, but it also means that the ICP ledger can never have the same internal representation as ICRC-1 ledgers. 
 
 In an attempt to comply with the ICRC-1 standard, the ICP ledger implements all endpoints defined in the ICRC-1 standard. However, this does not mean that the ICP ledger is an ICRC-1 ledger. The ICRC-1 standard clearly defines how accounts should be represented.
 The ICP ledger, due to its use of `AccountIdentifiers`, cannot fulfill this requirement. The ICP ledger successfully implements all ICRC-1 endpoints and they can be used for any other ICRC-1 endpoint. 
-Should a future ICRC-1 extension standard or the use of index canisters result in the exposure of the internal representation of accounts, any ICRC-1 ledger will deviate from the ICP ledger. 
\ No newline at end of file
+Should a future ICRC-1 extension standard or the use of index canisters result in the exposure of the internal representation of accounts, any ICRC-1 ledger will deviate from the ICP ledger. 
+
+## Index Canisters
+The ICP ledger lets you query created blocks through various endpoints. See the documentation on [how to interact with the ICP ledger for more info](./interact-with-ledger.md). If your ICRC-1 ledger supports the extension standard ICRC-3, then your ICRC-1 ledger will have an endpoint to serve transactions. 
+However, if you want to fetch transactions for a specific `Account` or `AccountIdentifier` you would have to first query all blocks/transactions from the ICP or an ICRC-1 compatible ledger and then search through them for transactions in which an `Account` or `AccountIdentifier` was involved in. To solve this issue index canisters exist. The index canisters allow for querying of transaction lists for a specific account. Furthermore, they offer endpoints for querying the current balance of an account and you can also fetch blocks from one of their endpoints. 
+
+There exist two different index canisters, one for the [ICP ledger](./icp-index-local-setup.md) and one for [ICRC-1 compatible ledgers](/docs/developer-docs/integrations/icrc-1/icrc1-index-setup.md). Due to the previously mentioned difference between the ICP ledger and ICRC-1 ledgers these two index canister will not be able to have identical endpoints. Since one uses `Account` and the other uses `AccountIdentifier` as an internal representation of accounts, the returned transactions will differ in this regard. 
```

Comment:
```suggestion
There are two different types of index canisters, one for the [ICP ledger](./icp-index-local-setup.md) and one for [ICRC-1 compatible ledgers](/docs/developer-docs/integrations/icrc-1/icrc1-index-setup.md). Due to the previously mentioned difference between the ICP ledger and ICRC-1 ledgers, these two index canister will not be able to have identical endpoints. Since one uses `Account` and the other uses `AccountIdentifier` as an internal representation of accounts, the returned transactions will differ in this regard. 
```